Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/e2e/Core-journeys/decisions_tests.cy.js

```diff
@@ -0,0 +1,59 @@
+import projectTaskList from "../../pages/projectTaskList";
+import { decisionPage } from '../../pages/decisionPage';
+import { Logger } from '../../support/logger'
+
+
+
+describe('Decisions Tests', () => {
+    beforeEach(() => {
+        Logger.log("Visit the homepage before each test");
+        projectTaskList.getHomePage();
+    });
+
+    it('Creating a conversion project and recording a new decision then editting the decision', () => {
+        Logger.log("Go to the home page then click create new conversion");
+        projectTaskList.getCreateNewConversion();
+
+        Logger.log("Click on create new conversion button on the next page");
+        projectTaskList.getCreateNewConversion();
+
+        Logger.log("Search and select the school, then click continue");
+        decisionPage.searchSchool('Manchester Academy (134224)').clickContinue();
+
+        Logger.log("Select no and continue on the next 3 pages regarding the school");
+        decisionPage.selectNoAndContinue();
+        decisionPage.selectNoAndContinue();
+        decisionPage.selectNoAndContinue();
+
+        Logger.log("Verify the selected school details");
+        decisionPage.assertSchoolDetails(
+            'Manchester Academy',
+            '134224',
+            'Manchester',
+            'Academy'
+        );
+
+        Logger.log("Click school details are correct, click continue");
+        decisionPage.clickContinue();
+
+        Logger.log("Search for the created project then select first one on the list");
+        decisionPage.clickOnFirstProject();
```

Comment:
Is this reliable enough? Should we not filter on the school first? Otherwise I could be creating a project in dev and break the cypress tests

Reply:
Yes, I understand the concern. I've been working on an improved logic where I use the created school ID and inject it into the URL. This way, we can be certain that we're working with the correct school. I'll push this implementation in the next test.